pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -298,14 +298,17 @@ func getConsoleTemplate(cr *api.KieApp) api.ConsoleTemplate {
 	}
 	template.Replicas = *cr.Status.Applied.Objects.Console.Replicas
 	template.Name = envConstants.App.Prefix
-	template.ImageURL = envConstants.App.Product + "-" + envConstants.App.ImageName + constants.RhelVersion + ":" + cr.Status.Applied.Version
+	cMajor, _, _ := MajorMinorMicro(cr.Status.Applied.Version)
```

Comment:
This is a method and should be named using a verb. e.g. `getMajorMinorMicro`

Reply:
done @ruromero